### PR TITLE
Add regression test for #317

### PR DESCRIPTION
**Pull request by darikg**

### CODE REVIEW

**Comment by amjith** [comment]
@j-bennet Can you check why the integration tests always fail the first time?

It seems like the Ctrl-D is the one that always fails. I'm not going to rerun the tests so you get a chance to review the failure.

File: tests/test_sqlcompletion.py [hunk 1]
```diff
@@ -454,3 +454,10 @@ def test_identifier_suggests_types_in_parentheses(text):
 def test_alias_suggests_keywords(text):
     suggestions = suggest_type(text, text)
     assert suggestions == [{'type': 'keyword'}]
+
+
+def test_invalid_sql():
+    # issue 317
+    text = 'selt *'
```
Comment (Abscond63) [inline tests/test_sqlcompletion.py]: Should not be alias for Selt,  bad handler for definition.

**Comment by j-bennet** [comment]
@amjith Looking.

**Comment by j-bennet** [comment]
Traceback for my reference:

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.9/bin/behave", line 11, in <module>
    sys.exit(main())
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/behave/__main__.py", line 109, in main
    failed = runner.run()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/behave/runner.py", line 672, in run
    return self.run_with_paths()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/behave/runner.py", line 693, in run_with_paths
    return self.run_model()
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/behave/runner.py", line 504, in run_model
    self.run_hook('after_all', self.context)
  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/behave/runner.py", line 405, in run_hook
    self.hooks[name](context, *args)
  File "features/environment.py", line 68, in after_all
    context.conf['pass'], context.conf['dbname'])
  File "/home/travis/build/dbcli/pgcli/tests/features/db_utils.py", line 70, in drop_db
    cr.execute('drop database if exists %s', (AsIs(dbname),))
psycopg2.OperationalError: database "pgcli_behave_tests_2_7_9" is being accessed by other users
DETAIL:  There are 1 other session(s) using the database.
```

Looks like features (test cases) are being run in parallel, so while this one is trying to create and drop db, other ones are trying to work with the same db. Will fix.

**Comment by amjith** [comment]
I'm going to merge this for now. I know the integration tests are failing. It'll be fixed in the future.

**Pull request merged**